content/redirect.js

```diff
@@ -20,9 +20,14 @@ function checkURLInvalidSymbols(url) {
   }
 }
 
+function isVanityRedirectUrl(url) {
```

Comment:
```suggestion
function isVanityRedirectURL(url) {
```
...is the convention. Mentioned in `docs/conventions.md`

Reply:
If you see more of these feel free to "upgrade".